frontend/src/components/planetSystem/PlanetSystem.scss

```diff
@@ -0,0 +1,348 @@
+@keyframes orbite_Mercure {
+  0% {
+    transform: rotate(180deg) translateY(10.5vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(10.5vw);
+  }
+}
+
+@keyframes orbite_Venus {
+  0% {
+    transform: rotate(180deg) translateY(15.5vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(15.5vw);
+  }
+}
+
+@keyframes orbite_Earth {
+  0% {
+    transform: rotate(180deg) translateY(21vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(21vw);
+  }
+}
+
+@keyframes orbite_Mars {
+  0% {
+    transform: rotate(180deg) translateY(26.5vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(26.5vw);
+  }
+}
+
+@keyframes orbite_Jupiter {
+  0% {
+    transform: rotate(180deg) translateY(37vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(37vw);
+  }
+}
+
+@keyframes orbite_Saturne {
+  0% {
+    transform: rotate(180deg) translateY(48vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(48vw);
+  }
+}
+
+@keyframes orbite_Uranus {
+  0% {
+    transform: rotate(180deg) translateY(61vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(61vw);
+  }
+}
+
+@keyframes orbite_Neptune {
+  0% {
+    transform: rotate(180deg) translateY(74.5vw);
+  }
+
+  100% {
+    transform: rotate(-180deg) translateY(74.5vw);
+  }
+}
+.système {
+  position: absolute;
+  left: 26vw;
```

Comment:
Using values in vw can break your design, another alternative seems more relevant.